resources/public/api/conf/1.0/application.yaml

```diff
@@ -103,22 +94,19 @@ paths:
             application/json:
               schema:
                 allOf:
-                - $ref: '#/components/schemas/individualPayeTestSupportSaEmploymentsAnnualSummaryResponse'
-              example:
-                employments:
-                - employerPayeReference: 123/AB456
-                  employerName: Company XYZ
-                  offPayrollWorkFlag: true
-                - employerPayeReference: 456/AB456
-                  employerName: Company ABC
-                  offPayrollWorkFlag: false
+                  - $ref: '#/components/schemas/individualPayeTestSupportSaBenefitsAnnualSummaryResponse'
```

Comment:
This shows the benefits response schema. Please update the schema to `individualPayeTestSupportSaEmploymentsAnnualSummaryResponse`